src/components/ActivityFeed/ActivityFeedItem.js

```diff
@@ -0,0 +1,530 @@
+            || ensRegistry[address]
+            || elipsizeAddress(address);
+        const isPPNTransaction = get(event, 'isPPNTransaction', false);
+        const information = event.accountType === ACCOUNT_TYPES.KEY_BASED ? 'Key wallet' : 'Smart wallet';
+        if (isPPNTransaction) {
+          data = {
+            name: usernameOrAddress,
+            avatarUrl,
+            customAddon: (
+              <TankAssetBalance
+                amount={`${directionSymbol} ${formattedValue} ${event.asset}`}
+              />
+            ),
+          };
+        } else {
+          data = {
+            name: usernameOrAddress,
+            information,
+            avatarUrl,
+            iconName: !avatarUrl ? directionIcon : null,
+            iconColor: isReceived ? '#497391' : 'negative',
+            itemValue: `${directionSymbol} ${formattedValue} ${event.asset}`,
+            valueColor: isReceived && formattedValue !== '0' ? 'positive' : 'text',
+          };
+        }
+    }
+    data.statusIcon = event.status === TX_PENDING_STATUS ? TX_PENDING_STATUS : '';
+    if (activeBlockchainNetwork === 'BITCOIN') {
+      data.information = 'Bitcoin wallet';
+    }
+    data.eventData = {
+      ...event,
+      value,
+      contact,
+      ...trxData,
+    };
+    return data;
+  }
+
+  getCollectibleTransactionEventData = (event: Object) => {
+    const isReceived = this.isReceived(event);
+    const { asset, icon } = event;
+    const usernameOrAddress = this.usernameOrAddress(event);
+    const information = `Collectible ${isReceived ? 'from' : 'to'} ${usernameOrAddress}`;
+    const contact = this.getContact(event);
+
+    return {
+      name: asset,
+      imageUrl: icon,
+      information,
+      status: isReceived ? STATUSES.RECEIVED : STATUSES.SENT,
+      statusColor: isReceived ? 'positive' : 'text',
+      eventData: { ...event, contact },
+    };
+  }
+
+  getBadgeRewardEventData = (event: Object): EventData => {
+    const { name, imageUrl } = event;
+    return {
+      name,
+      imageUrl,
+      information: 'Badge',
+      status: STATUSES.RECEIVED,
+      statusColor: 'positive',
+      eventData: { ...event },
+    };
+  }
+
+  getSocialEventData = (event: Object): ?EventData => {
+    const { onRejectInvitation, onAcceptInvitation } = this.props;
+    const { type, username, profileImage } = event;
+
+    let status;
+    if (type === TYPE_ACCEPTED) {
+      status = STATUSES.CONNECTED;
+    } else if (type === TYPE_SENT) {
+      status = STATUSES.REQUESTED;
+    } else {
+      status = null;
+    }
+
+    const data: EventData = {
+      name: username,
+      status,
+      avatarUrl: profileImage,
+      eventData: { ...event },
+      statusAsButton: type === TYPE_SENT,
+    };
+
+    if (type === TYPE_RECEIVED) {
+      data.information = 'Connection request';
+      data.rejectInvitation = () => createAlert(TYPE_REJECTED, event, () => onRejectInvitation(event));
```

Comment:
`rejectInvitation` and `acceptInvitation` are not being passed as functions, hence trying to accept or reject invite results in error: `rejectInvitation is not a function` or `acceptInvitation is not a function`